app/views/layouts/web/shared/_navbar.html.haml

```diff
@@ -59,5 +39,88 @@
                       = link_to article.title, article_path(article)
+                - if signed_in?
+                  .category-column
+                    .menu-title Программы и проекты
+                    %a{href: "#"}
+                      Программа «Лидер»
+                    %a{href: "#"}
+                      Программа «Inформация»
+                    %a{href: "#"}
+                      Программа «Добро»
+                    %a{href: "#"}
+                      Программа «Право»
+                    %a{href: "#"}
+                      Проект «Информационные технологии»
+                    %a{href: "#"}
+                      Проект «Диалог культур»
+                  - if @korporative_category.present?
+                    .category-column
+                      .menu-title Корпоративные проекты
+                      - @korporative_category.articles.presented.each do |article|
+                        = link_to article.title, article_path(article)
+                  .category-column
+                    .menu-title Библиотека
+                    %a{href: "#"}
+                      «Активаторы»
+                    %a{href: "#"}
+                      Газета «Этажи»
+                    %a{href: "#"}
+                      Методические разработки
+                - else
+                  .category-column
+                    .menu-title Деятельность
+                    %a{href: "#"}
+                      Профильные смены
+                    %a{href: "#"}
+                      Конкурс актива учащейся молодёжи «Я-лидер!»
+                    %a{href: "#"}
+                      Конкурс начинающих журналистов «Новый взгляд»
+                    %a{href: "#"}
+                      Марафон молодёжных инициатив
+                    %a{href: "#"}
+                      ALLMOL
+                    %a{href: "#"}
+                      Гражданиада
+                    %a{href: "#"}
+                      Уроки интернет-безопасности
+                    %a{href: "#"}
+                      «На старт!»
+                  .category-column
+                    .menu-title Программы и проекты
+                    %a{href: "#"}
+                      Программа «Лидер»
+                    %a{href: "#"}
+                      Программа «Inформация»
+                    %a{href: "#"}
+                      Программа «Добро»
+                    %a{href: "#"}
+                      Программа «Право»
+                    %a{href: "#"}
+                      Проект «Информационные технологии»
+                    %a{href: "#"}
+                      Проект «Диалог культур»
+                    %a{href: "#"}
+                      Корпоративная проекты
+                  .category-column
+                    .menu-title Команды МИЦ
+                    %a{href: "#"}
+                      г.Ульяновск
+                    %a{href: "#"}
+                      г.Димитровград
+                    %a{href: "#"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace